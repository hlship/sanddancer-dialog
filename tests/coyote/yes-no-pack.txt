(setup coyotes offer)
(now) (knock holds #strength)
w
(now) (turns in current room 6)
w
attack coyotes
say yes

