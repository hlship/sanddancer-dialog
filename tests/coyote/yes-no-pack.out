

but it's too late, you're crashing, you crash...

Middle of Nowhere (in the pickup truck)
You've smashed up against a tall Saguaro, which is all you can see out the
windshield except a few feet of desert sand, then blackness. Hell, you must have
driven off the road and crashed. Ocean keeps telling you this night commute on
this lonely highway's gonna kill you; maybe you should start listening.

On the dash you can see a piece of jade.

The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> (setup coyotes offer)
Query succeeded: (setup coyotes offer)
> (now) (knock holds #strength)
> teleport open desert

Open Desert
Clouds of dust swirl through the air, and you shiver in the chill night air.

Half-lost in the shadows, you see some dry bones, some clumps of sagebrush, and
a low bluff and that's about it.

> z
Nah, man, better keep busy or else it's back to brooding about your shit job.

It's over. They've caught you. They surround you, circling, growling, looking
for weakness, moving in for the kill.

> attack coyotes
It's over. The coyotes dive in for the kill, jaws snapping, and without hardly
realizing what you're doing you make a fist and slam it into the one that's
closest.

The coyote squeals, flung sideways by the strength of your punch, blood dripping
from its nose, and the rest of the pack pulls back in sudden caution. Another
coyote makes to lunge and you whirl towards it in fury, pulling back for another
blow, but the pack breaks in some collective decision and scatters, mangy
shadows slinking away into the desert. Except for one last coyote who stands his
ground.

And then you blink, and realize he's not a coyote after all. Just... just a guy.

He wears a couple of days of stubble, a scuffed and faded denim jacket, and,
ridiculously in the darkness, a pair of cheap sunglasses. He lights a cigarette,
then holds up a hand in annoyance to block your flashlight beam. "Turn that
thing off," he says, and either you do it or it goes off by itself, because the
next thing you know, it's darker and he's closer and the only light comes from
the red glow of the cigarette.

"That's better," he says. He takes a drag on the cigarette, then pulls another
from behind his ear and offers it to you. "Smoke?"

You could say yes or say no.

> say yes
It's freezing out and your nerves are jangled. "Sure, I could use a smoke" you
say.

"Out of smokes, eh?" he says, and hands you his last cigarette, and leans
forward for you to light up off him. His sunglasses reflect back nothing but
your own face weirdly lit by the glow from the cig, and for a moment it's like
the blackness behind them is deep, deeper than the sky, deeper than the
universe...

The coyote stares at you momentarily before gazing out at the horizon.
"Gotta keep an eye on the horizon," he says in a scratchy, slow voice. "Always
someone hunting. Border Patrol, National Guard, INS, ICE, Minutemen. You want to
help people get where they're going out here, you need more than strength. You
need to learn how to hide."

"I'm the Coyote," he says, "And we've been hiding from each other for a long
time."

You could ask for advice, ask who he is, ask about the rabbit, or ask about
trading.

> 
